src/safeds/ml/classical/classification/_gradient_boosting.py

```diff
@@ -13,27 +13,34 @@
 
 
 class GradientBoosting(Classifier):
-    """Gradient boosting classification.
+    """
+    Gradient boosting classification.
 
     Parameters
     ----------
     learning_rate : float
         The larger the value, the more the model is influenced by each additional tree. If the learning rate is too
         low, the model might underfit. If the learning rate is too high, the model might overfit.
+    number_of_trees: int
+        The number of boosting stages to perform. Gradient boosting is fairly robust to over-fitting so a large
+        number usually results on better performance.
 
     Raises
     ------
     ValueError
-        If `learning_rate` is non-positive.
+        If `learning_rate` is non-positive or the `number_of_trees` is less than or equal to 0.
     """
 
-    def __init__(self, learning_rate: float = 0.1) -> None:
+    def __init__(self, learning_rate: float = 0.1, number_of_trees: int = 100) -> None:
         # Validation
         if learning_rate <= 0:
             raise ValueError("The learning rate has to be greater than 0.")
+        if number_of_trees <= 0:
+            raise ValueError("The number of boosting stages to perform has to be greater than 0.")
```

Comment:
Swap order (same for remaining cases where `learning_rate` and `number_of_trees` appear.